Source/ASImageNode.mm

```diff
@@ -517,10 +517,19 @@ + (UIImage *)createContentsForkey:(ASImageNodeContentsKey *)key drawParameters:(
       key.willDisplayNodeContentWithRenderingContext(context, drawParameters);
       contextIsClean = NO;
     }
+    
+    UIColor *backgroundColor = key.backgroundColor;
+    UIColor *tintColor = key.tintColor;
+    
+    if (@available(iOS 13.0, *)) {
+      UITraitCollection *tempTraitCollection = [UITraitCollection traitCollectionWithUserInterfaceStyle:key.userInterfaceStyle];
```

Comment:
`ASGraphicsCreateImageWithTraitCollectionAndOptions` runs drawing code inside `[UITraitCollection -performAsCurrentTraitCollection]` thus we don't need to resolve `tintColor` here

Reply:
If you don't resolve ,  ASDisplayNode & ASImageNode will use wrong color, when overrideUserInterfaceStyle is set

Reply:
What happens if `overrideUserInterfaceStyle` is correctly picked up by the node? Will we still need this change? And why?